modules/dev-tools/deprecation.php

```diff
@@ -190,8 +203,60 @@ public function deprecated_function( $function, $version, $replacement = '', $ba
 			$print_deprecated = true;
 		}
 
+		return $print_deprecated;
+	}
+
+	/**
+	 * Deprecated Function
+	 *
+	 * Handles the deprecation process for functions.
+	 *
+	 * @param $function
+	 * @param $version
+	 * @param string $replacement
+	 * @param null $base_version
+	 */
+	public function deprecated_function( $function, $version, $replacement = '', $base_version = null ) {
+		$print_deprecated = $this->check_deprecation( $function, $version, $replacement, $base_version );
+
 		if ( $print_deprecated ) {
 			_deprecated_function( $function, $version, $replacement );
 		}
 	}
+
+	/**
+	 * Deprecated Hook
+	 *
+	 * Handles the deprecation process for hooks.
+	 *
+	 * @param $hook
+	 * @param $version
+	 * @param string $replacement
+	 * @param string $message
+	 * @param null $base_version
+	 */
+	public function deprecated_hook( $hook, $version, $replacement = '', $message = '', $base_version = null ) {
```

Comment:
Since we want a unique format, remove the `message` argument. The same goes for `deprecated_argument`

Reply:
Done.